### ISSUE TITLE
Clarify acceptable lead times for event-based requests

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/team-compass#740]
### Context

We need to provide better communication around planning for events where communities want to use the hubs we provide and require either monitoring or increase in resources. We often receive requests to support events with short notice and this is not sustainable as it causes a scramble to find people to manage the scale-up/-down and the event time itself.

### Proposal

We need two things:

- an acceptable deadline to submit a request for an event, e.g. two weeks before it will take place (this is just a suggestion, we should decide this as a team)
  - and actually I believe if the community is organising the event themselves, then there's no reason why they shouldn't be able to begin having the discussion with us at the point of date selection during planning
- a consequence for missing the deadline, which could be one of two options (there may be more options, and I'm happy to hear them):
  - declining the request (I think we, as a team, need to get better at saying no)
  - charging a surcharge for the extra effort to support the event in the reduced lead time

This should be _as well as_ any events charging we already do (e.g. the exams for UToronto that we have prototyped).

Once we have these policies established, we should _clearly communicate them somewhere_. Maybe a "how to schedule an event on your 2i2c hub" guide in docs.2i2c.org. It may also be beneficial to use another freshdesk form that collects relevant info and embed that into the page and flags them in a different way to our usual support requests.

**Comment by sgibson91** [comment]
Update: we do three weeks notice here https://docs.2i2c.org/community/events/#notify-the-2i2c-team-about-the-event

But I still think we need to promote this better

**Comment by choldgraf** [comment]
We do have an events and workshops guide in the service docs here:

https://docs.2i2c.org/community/events/#notify-the-2i2c-team-about-the-event

That covers things like "how much lead time to give". So maybe the primary problem here is to make sure that communities know about these guidelines, and that we hold them accountable for following them.

I don't know if that means the information needs to be in a different place, or more explicit, or consequences more clear, etc. But clearly we have an "unnoticed information" problem to overcome.

**Comment by sgibson91** [comment]
I also don't think that web page outlines the _consequences_ of missing the three week period at all, so we should add that

**Comment by choldgraf** [comment]
Good idea @sgibson91 . I've added a tasklist in the top comment and added that to one of the items

**Comment by choldgraf** [comment]
### Could we use a freshdesk widget for this?

If we used a dedicated "events" FreshDesk widget, perhaps we could both structure the responses from the person more effectively, and include a link in the widget like "I have read and agreed to [the 2i2c events service requirements](https://docs.2i2c.org/community/events/#notify-the-2i2c-team-about-the-event)

**Comment by sgibson91** [comment]
I think a freshdesk widget _in theory_ is a good idea, but I saw conversation in Slack the other day that the current one for support is also not seeing the traction we would like. But maybe that's not an argument against implementation so much as an argument that we clearly needs a comms strategy.

**Comment by choldgraf** [comment]
Hmm. I think this will require coordination between @jmunroe (representing the perspective of communities he's connecting w/) and the engineering team (representing the support system, cc @damianavila). We clearly need a better system that is more reliably used, and in a more sustainable and predictable way for our team.
